apps/vaporgui/ParticleEventRouter.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "RenderEventRouterGUI.h"
+#include "vapor/ParticleRenderer.h"
+
+//! \class ParticleEventRouter
+//! \ingroup Public_GUI
+//! \brief Particle renderer GUI
+//! \author Stas Jaroszynski
+
+class ParticleEventRouter : public RenderEventRouterGUI {
+public:
+    ParticleEventRouter(QWidget *parent, VAPoR::ControlExec *ce);
+    static string GetClassType() { return VAPoR::ParticleRenderer::GetClassType(); }
+    string        GetType() const { return GetClassType(); }
+    bool          Supports2DVariables() const { return true; }
```

Comment:
Should Supports{2D,3D} be returning true? This methods refers to a variable's **topolological** (Euclidean) dimension: E.g. zero for points, one for lines, two for planes, and so on.

Reply:
Fixed.